cvFit/src/app/home/welcome.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+
+@Component({
+  templateUrl: './welcome.component.html',
+  styleUrls: ['./welcome.component.css']
+})
+export class WelcomeComponent {
+  public pageTitle = 'Home';
+  public longTextChris = 'Need help getting started in your body building journey?  This is the place to be!  The team is inteligent, helpful, and personable.  They helped me to become the person I am today.  I would highly recommend the in person training sessions as well as making use of all the awesome tools the team provides via the website';
```

Comment:
Unless you plan to pull this data from an API, you should try to put text in HTML. It helps with later features like translation.

Reply:
Sounds good, I will move it to the html.  Have strings like this in other places as well that will be moved